src/main/java/school/hei/haapi/service/utils/DataFormatterUtils.java

```diff
@@ -32,4 +32,8 @@ public static String numberToWords(int number) {
     return intConverter.asWords(number).toUpperCase();
   }
 
+  public static String handleNull(String toHandle) {
+    return toHandle == null ? "" : toHandle.toLowerCase();
```

Comment:
This does not work when a course have a null attribute in its. You can add this test in your CourseIT and try it yourself. 

Refer to BPartners customers filter to know more.